playwright/_impl/_connection.py

```diff
@@ -260,11 +263,8 @@ def _dispatch(self, msg: ParsedMessagePayload) -> None:
                     g.switch(self._replace_guids_with_channels(params))
             else:
                 object._channel.emit(method, self._replace_guids_with_channels(params))
-        except Exception:
-            print(
```

Comment:
lets keep the print statement, since not every event emit is followed by a Playwright call IMO.

Reply:
Fixed

Reply:
Lets re-add the print statement please, being more verbose here is better.

Reply:
That would clutter up the console in Pytest, I don't think print is needed as the exception is fixed is now correctly handled

Reply:
Imagine you have an event handler, which throws but you never do something Playwright related afterwards. With your current impl this would then never show the error from my understanding.

Reply:
Added